src/components/board/board.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import "./board.css";
+import Square from "../square/square";
+
+const FILES = ["a", "b", "c", "d", "e", "f", "g", "h"];
+const RANKS = [1, 2, 3, 4, 5, 6, 7, 8];
+
+function makeBoard(ranks, files) {
+  return ranks.reduce((board, rank) => {
+    files.forEach(file => {
+      const coordinate = `${file}${rank}`;
+      board.push({ coordinate: coordinate, piece: null});
```

Comment:
why coordinate? i like id so much better.

Reply:
it's really up to you - as long as you understand that "coordinate" is the unique identity of the object then it's fine with me. everything has an id property usually :)

Reply:
Typically everything has an id and the distinguishing thing is the item that has the id - like Sqaure.id or Piece.id. It is up to you, as long as you can keep it straight. 

Keep in mind, when naming things like this, you may move to another project and another developer may have to take this over. So you want to make it descriptive enough so someone brand new knows what it means. So you have to think beyond while still focusing on the present. Dang - that's deep. :-)